### DISCUSSION TITLE
Error: P3009 migrate found failed migrations in the target database, new migrations will not be applied.

### DISCUSSION THREAD

**Discussion by abhay44** [prisma/orm#20800] [Q&A]
### Question

I created a new database on AWS RDS by restoring to point in time of an existing db. On kubernetes cluster `npx prisma migrate deploy` is throwing error and logs are empty. Even in the table `_prisma_migrations` there are no logs. Can anyone help on the same?

### How to reproduce (optional)

1. Run npx prisma migrate deploy
2. Error: P3009
```
migrate found failed migrations in the target database, new migrations will not be applied. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve
The `20210116185514_initial` migration started at 2023-08-22 13:18:45.536926 UTC failed with the following logs:

```

- OS: Ubuntu
- Database: PostgreSQL
- Node.js version: 14.17
- Prisma Version - 4.0.0

**Comment by janpio** [comment]
What was the output on the original execution of `migrate deploy`?
But the failed migration is recorded in `_prisma_migrations`, correct?

**Reply by leppaott** [reply]
We too have a failing migration without any error:

```
                  id                  |                             checksum                             | finished_at |          migration_name           | logs | rolled_back_at |          started_at           | applied_steps_count 
--------------------------------------+------------------------------------------------------------------+-------------+-----------------------------------+------+----------------+-------------------------------+---------------------
 5aae4f01-9153-48b5-bc15-6e96dba7fe51 | 05d7a99bd39324a6e5466dbec89897e85b1b4efb5c24d16ab0cf9480803bb029 |             | name_of_migration |      |                | 2024-01-18 07:05:32.672378+00 |                   0
 ```
 
I guess we have to look at PG logs.

**Reply by temrb** [reply]
same issue here when migrate deploying prod

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!

**Comment by n3crongam3s** [comment]
I haved this error and solved by running "prisma migrate reset"  i hope this can help many peeple
